build/gulp/renovation-pipes.js

```diff
@@ -22,7 +23,7 @@ module.exports = {
     replaceWidgets: lazyPipe()
         .pipe(function() {
             return gulpIf(isOldComponentRenovated, gulpEach((content, file, callback) => {
-                const component = renovatedComponents.find(component => component.name.toLowerCase() === file.stem);
+                const component = renovatedComponentsMeta.find(component => component.fileName === file.stem);
```

Comment:
May be transform renovatedComponentsMeta to static object with componentFileNameMap = { [component.fileName]: component}. And replace code to
```
const component = componentFileNameMap[file.stem]
```